docs/intro/intro_bqm.rst

```diff
@@ -0,0 +1,96 @@
+.. _intro_bqm:
+
+=======================
+Binary Quadratic Models
+=======================
+
+dimod provides a :term:`binary quadratic model` (BQM) class that encodes
+:term:`Ising` and quadratic unconstrained binary optimization (\ :term:`QUBO`\ )
+models used by samplers such as the D-Wave system.
+
+For an introduction to BQMs, see :std:doc:`Binary Quadratic Models <oceandocs:concepts/bqm>`.
+
+dimod can represent BQMs with classes suited to different needs for
+performance and ease of use. Beginners might start with the simple Python
+:ref:`AdjDictBQM <adjdictbqm_dimod>` class (:class:`dimod.BinaryQuadraticModel`),
+and later switch to a higher-performing class with a C++ implementation.
+
+For descriptions of all supported BQM representations, see :ref:`bqm`.
+
+BQM Generation
+==============
+
+The small four-node `maximum cut <https://en.wikipedia.org/wiki/Maximum_cut>`_
+problem shown in this figure,
+
+.. figure:: ../_images/four_node_star_graph.png
+    :align: center
+    :scale: 40 %
+    :name: four_node_star_graph
+    :alt: Four-node star graph
+
+    Star graph with four nodes.
+
+Can be represented, as shown in the
+`dwave-examples <https://github.com/dwave-examples/maximum-cut>`_ Maximum Cut
+example, by a QUBO:
+
+.. math::
+
+   Q = \begin{bmatrix} -3 & 2 & 2 & 2\\
+                        0 & -1 & 0 & 0\\
+                        0 & 0 & -1 & 0\\
+                        0 & 0 & 0 & -1
+       \end{bmatrix}
+
+As mentioned above, for learning and testing with small BQMs, dimod's Python dict
+representation of BQMs is convenient:
+
+>>> qubo = dict({(0, 0): -3, (1, 1): -1, (0, 1): 2, (2, 2): -1,
```

Comment:
Would just doing `qubo = {(0, 0): -3, (1, 1): -1, (0, 1): 2, (2, 2): -1, ...}` be simpler?